src/test/java/ru/l3r8y/prefixed/PrefixedCopTest.java

```diff
@@ -56,10 +67,39 @@ void printsLogs(@Mktmp final Path temp) throws IOException {
                     "Lints URL was not printed, but it should",
                     f.log().content(),
                     Matchers.containsString(
-                        "[INFO] Boom!"
+                        "[INFO] Enforcing @Prefixed naming conventions in:"
                     )
                 );
             }
         );
     }
+
+    @Test
+    @Disabled
+    void failsOnWrongOrEmptyPrefixes(@Mktmp final Path temp) throws Exception {
+        final byte[] interfaze = new BytesOf(new ResourceOf("UnderTestInterface.java")).asBytes();
```

Comment:
@l3r8yJ let's inline these redundant variables to `f.files(..).file(..).write()`